docs/reference/configuration.rst

```diff
@@ -278,6 +284,14 @@ to an image to use as the background for the installer. As with ``splash``, the
 path should *exclude* the extension, and a platform-appropriate extension will
 be appended when the application is built.
 
+``long_description``
+~~~~~~~~~~~~~~~~~~~~
+
+A longer description of the purpose of the application. This description can be
+multiple paragraphs, if necessary. The long description *must not* be a copy of
+the ``description``, or include the ``description`` as the first line of the
+``long_description``.
```

Comment:
Breaking these rules didn't stop me from building a Debian package. Are they required by one of the other packaging tools? If not, it might be more useful to simply state that the `long_description` will appear below the `description`, and let the user decide how they want to split the content between them.

Reply:
Debian throws a linting error if the [long description is a copy of the description](https://lintian.debian.org/tags/description-synopsis-is-duplicated). That won't stop the package from being built (or even installed, AFAIK), but it will break if you try to submit it anywhere.